### PR TITLE
YC-609: Improve backoffice tabs

### PR DESCRIPTION
**Pull request by rbovard**
Fix https://jira.liip.ch/browse/YC-609

Please review.

### CODE REVIEW

File: permits/templates/permits/_permit_request_actions.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h2>{% trans "" %}</h2>
       {% if forms.amend %}
 
           <li class="nav-item">
-              <a class="btn btn-primary btn-action nav-link{% if active_form == "amend" %} active{% endif %}" id="amend-tab" data-toggle="tab" href="#amend" role="tab" aria-controls="amend" aria-selected="{% if active_form == "amend" %}true{% else %}false{% endif %}">{% trans "Traitement" %}</a>
+              <a class="btn btn-primary btn-action nav-link{% if active_form == "amend" %} active{% endif %}" id="amend-tab" data-toggle="tab" href="#amend" role="tab" aria-controls="amend" aria-selected="{% if active_form == "amend" %}true{% else %}false{% endif %}">{% trans "Général" %}</a>
```
[the remark this answers is not in the document]
Comment (monodo) [reply permits/templates/permits/_permit_request_actions.html:14]: Well "Traitement" is also not perfect...

**Review by monodo — CHANGES_REQUESTED** [review]

File: permits/templates/permits/_permit_request_actions.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (rbovard) [reply permits/templates/permits/_permit_request_actions.html:14]: Yes, it was the reason I tried to replace "Traitement" with something...

Any suggestion?

[the remark this answers is not in the document]
Comment (rbovard) [reply permits/templates/permits/_permit_request_actions.html:14]: About the action, a pilot has no obligation to go in this tab, the important one is the send the permit to validation.

[the remark this answers is not in the document]
Comment (rbovard) [reply permits/templates/permits/_permit_request_actions.html:14]: If you prefer, I can keep Traitement (even if I think Général is better).

**Review by monodo — APPROVED** [review]

**Pull request merged**